### ISSUE TITLE
Allow policy actions to be defined via class methods only

### ISSUE THREAD

**Issue by chrisalley** [pundit-community/pundit.js#43]
Following the discussion in https://github.com/johno/pundit/pull/41, it would be simpler to be able to define actions without explicitly defining an actions map, e.g.

```typescript
class PostPolicy extends Policy {
  user: AuthorisableUser
  record: AuthorisablePost

  constructor(user: AuthorisableUser, record: AuthorisablePost) {
    super(user, record)
  }

  view(): boolean {
    return true
  }

  publish(): boolean {
    return this.user.id === this.record.userId
  }

  destroy(): boolean {
    return this.user.isAdmin
  }
}
```

**Comment by johno** [comment]
100% agree there!

**Comment by chrisalley** [comment]
Solved in #48 and #49.
